Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -616,8 +631,8 @@ def handle_proxy(proxy_param_name='proxy', checkbox_default_value=False, handle_
     proxies = {}  # type: dict
     if demisto.params().get(proxy_param_name, checkbox_default_value):
         proxies = {
-            'http': os.environ.get('HTTP_PROXY') or os.environ.get('http_proxy', ''),
-            'https': os.environ.get('HTTPS_PROXY') or os.environ.get('https_proxy', '')
+            'http': add_http_prefix_if_missing(os.environ.get('HTTP_PROXY') or os.environ.get('http_proxy', '')),
+            'https': add_http_prefix_if_missing(os.environ.get('HTTPS_PROXY') or os.environ.get('https_proxy', ''))
```

Comment:
If I understand correctly, in this function you should also call `ensure_proxy_has_http_prefix` to make sure to modify the env variables.

Reply:
done